src/ontodia/diagram/linkView.ts

```diff
@@ -60,14 +64,28 @@ export class LinkView extends joint.dia.LinkView {
         }
 
         const showLabels = typeModel && typeModel.get('showLabel');
-        const labelAttributes = showLabels ? [{
-            position: 0.5,
-            attrs: {text: {
-                text: this.view.getLinkLabel(linkTypeId).text,
-            }},
-        }] : [];
-
-        merge(linkAttributes, {labels: labelAttributes});
+        const labelAttributes: joint.dia.LinkLabelAttributes[] = [];
+
+        if (showLabels) {
+            const label = {...style.label};
+            const value = _.get<LocalizedString[]>(style, 'label.attrs.text.text');
```

Comment:
Please, don't use `lodash.get` in TypeScript code, because then we lose the static typing benefits.